browser/src/dom/Draggable.js

```diff
@@ -21,6 +21,18 @@ L.Draggable = L.Evented.extend({
 		}
 	},
 
+	_manualDrag: function() {
+		return false;
```

Comment:
This is always false here. I see no places where it will be true. Is that expected? Or for future flexibility and extension?